### PR TITLE
PLAT-1173 Update CLI

### PR DESCRIPTION
**Pull request by justin-shipstation**
Updated the CLI with the latest version of the SDK and the Loader. 

- Removed originCountries, destinationCountries
- Added availableCountries

### CODE REVIEW

File: src/core/test-app/tests/create-shipment-international.ts [hunk 1]
```diff
@@ -90,9 +90,10 @@ export class CreateShipmentInternational extends Suite {
     let shipTo;
     try {
       [shipFrom, shipTo] = useInternationalShipmentAddresses(this.deliveryService);
-    } catch {
+    } catch(error) {
```
Comment (lzhang) [inline src/core/test-app/tests/create-shipment-international.ts:93]: ?

File: src/core/test-app/tests/create-shipment-international.ts [hunk 2]
```diff
@@ -90,9 +90,10 @@ export class CreateShipmentInternational extends Suite {
...
       return undefined;
     }
+    console.log(shipFrom, shipTo);
```
Comment (lzhang) [inline src/core/test-app/tests/create-shipment-international.ts:96]: ?

File: src/core/test-app/tests/create-shipment-international.ts (hunk 1 above)
Comment (justin-shipstation) [reply src/core/test-app/tests/create-shipment-international.ts:93]: remnants of when I was trying to figure out why the tests were failing.

File: src/core/test-app/tests/create-shipment-international.ts (hunk 2 above)
Comment (justin-shipstation) [reply src/core/test-app/tests/create-shipment-international.ts]: Cleaned up.

File: src/core/test-app/utils/find-matching-origin-and-destination-countries.ts [hunk 3]
```diff
@@ -1,51 +1,28 @@
 import { Country, DeliveryService } from "@shipengine/connect-sdk";
```
Comment (rkrauskopf) [inline src/core/test-app/utils/find-matching-origin-and-destination-countries.ts:1]: @justin-shipstation Rename this file to `find-matching-available-countries`

Comment (justin-shipstation) [reply src/core/test-app/utils/find-matching-origin-and-destination-countries.ts]: Updated

**Review by rkrauskopf — APPROVED** [review]

**Review by rkrauskopf — APPROVED** [review]

**Pull request merged**